src/main/resources/application.yml

```
@@ -0,0 +1,22 @@
+  web:
+    resources:
+      add-mappings: false
+
+server:
+  port: 8080
+  application:
+    name: "travel-journal-ai"
+
+ai:
+  server:
+    url: http://localhost:8081/api/diary/generate
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)